recipes/gnu-getopt/all/conanfile.py

```diff
@@ -0,0 +1,76 @@
+import os
+
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.env import VirtualBuildEnv
+from conan.tools.files import copy, get
+from conan.tools.gnu import Autotools, AutotoolsToolchain
+from conan.tools.layout import basic_layout
+from conan.tools.microsoft import is_msvc
+
+required_conan_version = ">=1.54.0"
+
+
+class GnuGetoptConan(ConanFile):
+    name = "gnu-getopt"
+    description = "GNU getopt(1) command-line utility"
+    license = "GPL-2.0-or-later"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://frodo.looijaard.name/project/getopt"
+    topics = ("gnu", "getopt", "utility", "command-line", "parsing")
+
+    package_type = "application"
+    settings = "os", "arch", "compiler", "build_type"
+
+    @property
+    def _settings_build(self):
+        return getattr(self, "settings_build", self.settings)
+
+    def configure(self):
+        self.settings.rm_safe("compiler.cppstd")
+        self.settings.rm_safe("compiler.libcxx")
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    def package_id(self):
+        del self.info.settings.compiler
+
+    def validate(self):
+        if is_msvc(self):
+            # Requires POSIX headers and libs
+            raise ConanInvalidConfiguration("MSVC is not supported. Consider using MSYS2 instead.")
+
+    def build_requirements(self):
+        if self._settings_build.os == "Windows":
+            self.win_bash = True
+            if not self.conf.get("tools.microsoft.bash:path", check_type=str):
+                self.tool_requires("msys2/cci.latest")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def generate(self):
+        env = VirtualBuildEnv(self)
+        env.generate()
+
+        tc = AutotoolsToolchain(self)
+        # liblastlog support requires sqlite3
```

Comment:
Any insight into not adding this as an option? I'm happy to keep as-is, but wanted to hear some of your insight if you had any :)

Reply:
We almost certainly shouldn't add any extra libraries to avoid potential conflicts with `libmount` and `util-linux-libuuid`.

Reply:
As far as I can understand the project setup of `util-linux`, the `getopt` executable does not even make use of the liblastlog dependency anyway.